systems/WikibaseClient/10-Quality.md

```diff
@@ -1 +1,25 @@
 # Quality Requirements
+
+## Quality Tree
+
+![Quality Tree](./diagrams/09-quality-tree.drawio.svg)
+
+## Quality Scenarios
+
+### Testability
+
+The PHP code is covered by integration and unit tests in [PHPUnit](https://phpunit.readthedocs.io/en/9.5/).  
+Tha JavaScript, TypeScript and Vue code is covered by unit, integration and e2e tests in [Jest](https://jestjs.io/docs/en/getting-started.html).  
+UIs such as Data Bridge are covered by browser tests via [wdio](https://webdriver.io/docs/gettingstarted).
+
+Test coverage statistics for Data Bridge are provided via Jest but that functionality does not run on CI.  
+Test coverage for PHP code is measured via [SonarQube](https://www.sonarqube.org/)
+
+### Interoperability
+
+Wikibase Client provides several Action API modules. For more info see the [API Building Blocks section](./05-Building_Block_View.md#apis).  
+Wikibase Client also provides the so called Special Pages. For more see the [Special Pages Building Blocks section](./05-Building_Block_View.md#special-pages).
+
+### Efficiency
+
+As mentioned in section [Quality Goals](./01-Introduction_and_Goals.md#quality-goals), Entity data which is included in a Wikibase Client's content is eventually consistent with updates on the corresponding Wikibase Repository without any actions being performed by the reader/editor.
```

Comment:
Hm, I don't see much value in repeating the quality goal we have in section 1 here again. I'm also unsure how it relates to efficiency. *Data consistency* is the quality attribute that this goal refers to.